algebra/src/bw6_761/curves/g1.rs

```diff
@@ -0,0 +1,108 @@
+use crate::{
+    biginteger::{BigInteger384, BigInteger768},
+    bw6_761::{Fq, Fr},
+    curves::{
+        models::{ModelParameters, SWModelParameters},
+        short_weierstrass_jacobian::{GroupAffine, GroupProjective},
+    },
+    field_new,
+};
+
+pub type G1Affine = GroupAffine<Parameters>;
+pub type G1Projective = GroupProjective<Parameters>;
+
+#[derive(Clone, Default, PartialEq, Eq)]
+pub struct Parameters;
+
+impl ModelParameters for Parameters {
+    type BaseField = Fq;
+    type ScalarField = Fr;
+}
+
+impl SWModelParameters for Parameters {
+    /// COEFF_A = 0
+    #[rustfmt::skip]
+
+    const COEFF_A: Fq = field_new!(Fq, BigInteger768([0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]));
+
+    /// COEFF_B = 6891450384315732539396789682275657542479668912536150109513790160209623422243491736087683183289411687640864567753786613451161759120554247759349511699125301598951605099378508850372543631423596795951899700429969112842764913119068298
+    #[rustfmt::skip]
+    const COEFF_B: Fq = field_new!(Fq, BigInteger768([
+        0xf29a000000007ab6,
+        0x8c391832e000739b,
+        0x77738a6b6870f959,
+        0xbe36179047832b03,
+        0x84f3089e56574722,
+        0xc5a3614ac0b1d984,
+        0x5c81153f4906e9fe,
+        0x4d28be3a9f55c815,
+        0xd72c1d6f77d5f5c5,
+        0x73a18e069ac04458,
+        0xf9dfaa846595555f,
+        0xd0f0a60a5be58c,
+    ]));
+
+    /// COFACTOR =
+    /// 26642435879335816683987677701488073867751118270052650655942102502312977592501693353047140953112195348280268661194876
+    #[rustfmt::skip]
+    const COFACTOR: &'static [u64] = &[
+        0x3de580000000007c,
+        0x832ba4061000003b,
+        0xc61c554757551c0c,
+        0xc856a0853c9db94c,
+        0x2c77d5ac34cb12ef,
+        0xad1972339049ce76,
+    ];
+
+    /// COFACTOR^(-1) mod r =
+    /// 91141326767669940707819291241958318717982251277713150053234367522357946997763584490607453720072232540829942217804
+    #[rustfmt::skip]
+    const COFACTOR_INV: Fr = field_new!(Fr, BigInteger384([
+        489703175600125849,
+        3883341943836920852,
+        1678256062427438196,
+        5848789333018172718,
+        7127967896440782320,
+        71512347676739162,
+    ]));
+
+    /// AFFINE_GENERATOR_COEFFS = (G1_GENERATOR_X, G1_GENERATOR_Y)
+    const AFFINE_GENERATOR_COEFFS: (Self::BaseField, Self::BaseField) =
+        (G1_GENERATOR_X, G1_GENERATOR_Y);
+}
```

Comment:
We probably want to specialize the computation in the curve equation to take advantage of the fact that `a == 0`. (ie here: https://github.com/scipr-lab/zexe/blob/a67131c18490587c4b3d811ce27eb8619e711b48/algebra-core/src/curves/models/mod.rs#L22)
```suggestion
    #[inline(always)]
    fn mul_by_a(elem: &Self::BaseField) -> Self::BaseField {
    	use crate::Zero;
        Self::BaseField::zero()
    }
}
```